diff --git a/index.d.ts b/index.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..0dc8d1f81a0577152959fdb1332c91639d0227e9
--- /dev/null
+++ b/index.d.ts
@@ -0,0 +1,20 @@
+export interface CompareValue {
+  key: string;
+  value: unknown;
+}
+
+export type CompareFunction = (
+  first: CompareValue,
+  second: CompareValue
+) => number;
+
+export interface StringifyOptions {
+  cmp?: CompareFunction;
+  cycles?: boolean;
+  space?: string | number;
+}
+
+export default function stringify(
+  data: unknown,
+  options?: StringifyOptions | CompareFunction
+): string;
